demo-playground/src/main/java/com/google/android/flexbox/recyclerview/FlexItemAdapter.java

```diff
@@ -63,6 +68,7 @@ public void onBindViewHolder(FlexItemViewHolder holder, int position) {
                 new FlexItemChangedListenerImplRecyclerView(mLayoutManager, this),
                 adapterPosition));
         holder.bindTo(mLayoutParams.get(position));
+        Log.d(TAG, "onBindViewHolder. Holder: " + holder + ", Position: " + position);
```

Comment:
It's shown only in the demo app, but used only for the development purpose. Deleted.